src/js/grapl-cdk/swarm/swarm_setup.py

```diff
@@ -239,15 +238,14 @@ def _join_worker_nodes(
     ssm: Any,
     cloudwatch: CloudWatchClient,
     prefix: str,
-    instances: Iterator[InstanceTuple],
+    instances: Sequence[InstanceTuple],
```

Comment:
is this Sequence bc it's convenient, or is ordering important?  (contrasting to Collection)

Reply:
Sequence is the closest parent of both List and Tuple. Collection also incldues like sets and dict and stuff iirc